src/box/key_def.h

```diff
@@ -387,7 +389,7 @@ typedef union PACKED {
 		 *
 		 * => key: ["bar"]
 		 *
-		 * Note: When the path is given, <field_type>
+		 * Note: When the path is given, \a field_type
 		 * means type of the nested field.
```

Comment:
The weird thing is that neither the fields on the structure, nor the comments to them are placed into the documentation. In fact, it means that one can't use the API based only on the doc. Need to look into module.h anyway.

Don't know, is there a way to instruct Doxygen to don't strip the documentation.

Reply:
Filed #8513.